contracts/Vault.sol

```diff
@@ -107,6 +109,22 @@ contract Vault is
     // The total of principal deposited
     uint256 public totalPrincipal;
 
+    modifier onlyManager() {
+        require(
+            hasRole(MANAGER_ROLE, msg.sender),
+            "Vault: caller is not manager"
+        );
+        _;
+    }
+
+    modifier onlyAdmin() {
+        require(
+            hasRole(DEFAULT_ADMIN_ROLE, msg.sender),
```

Comment:
why not just use the default `onlyRole` modifiers? this seems unnecessary

Reply:
onlyRole revert with too long strings.
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/access/AccessControl.sol#L109

Reply:
true, but is that a problem for us? it costs extra gas, but it shouldn't affect us, only wrong transactions, right?

Reply:
fixed